specs/lib/loadGltfUrisSpec.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+var fs = require('fs');
+var loadGltfUris = require('../../lib/loadGltfUris');
+var fragmentShaderPath = './specs/data/boxTexturedUnoptimized/CesiumTexturedBoxTest0FS.glsl';
+
+describe('loadGltfUris', function() {
+    it('loads an external shader', function(done) {
+        var gltf = {
+            "shaders": {
+                "CesiumTexturedBoxTest0FS": {
+                    "type": 35632,
+                    "uri": "CesiumTexturedBoxTest0FS.glsl"
+                }
+            }
+        };
+        
+        fs.readFile(fragmentShaderPath, function (err, fragmentShaderData) {
```

Comment:
This can be done in a `beforeAll` and assign it to a variable scoped for the lifetime of the test suite, e.g., 
```javascript
describe('loadGltfUris', function() {
    var fragmentShaderData;

    beforeAll(/* ... */);

    it(/* ... */);
    it(/* ... */);
    // ...
});